test/IRGen/builtins_objc.swift

```diff
@@ -0,0 +1,35 @@
+// RUN: %target-swift-frontend -parse-stdlib -primary-file %s -emit-ir | %FileCheck %s
+
+// REQUIRES: executable_test
+// REQUIRES: objc_interop
+
+import Swift
+import Foundation
+
+@objc enum ObjCEnum: Int32 { case X }
+@objc class ObjCClass: NSObject {}
+class NonObjCClass {}
+
+@_silgen_name("use")
+func use(_: Builtin.RawPointer)
+
+
+func getObjCTypeEncoding<T>(_: T) {
+  // CHECK: call void @use({{.* i8]\*}} [[INT32:@[0-9]+]],
```

Comment:
I could, but I wasn't sure that the churn of tracking down and conditionalizing the exact encode strings for each type on each supported platform really gave meaningful coverage over just verifying that the builtin works at all. Clang ought to exercise the underlying type encoding code fairly well already.